SVProfilePictureViewer/PictureViewerForm.cs

```diff
@@ -84,17 +89,17 @@ enum MaskType {
         return Color.FromArgb(255, r, g, b);
       }
 
-      Bitmap ExtractComponent(int offset, MaskType maskType = MaskType.None) {
+      Bitmap ExtractComponent(int offset, MaskType maskType = MaskType.None, int spacing = 4) {
```

Comment:
I understand why the offset was 4 but I just assumed that the masks corresponded each on a single image, that's why I added the parameter, so that both masks could be used in combination of each image

Reply:
That is my mistake then since I was going off that you had the light images m1 and m2 which looked like you were trying to get the mask as their RGB values.
If what you said is the case then change the name to something like maskOffset.